### PR TITLE
securing admin

### PR DESCRIPTION
**Pull request by vallettea**

### CODE REVIEW

File: core/server/admin.js [hunk 1]
```diff
@@ -105,7 +105,10 @@ app.use("/Admin", express.static(path.join(__dirname, '../clients/Admin')));
 app.use("/_common", express.static(path.join(__dirname, '../clients/_common')));
 
 app.get('/', function(req, res){
-    res.sendFile(path.join(__dirname, '../clients/Admin/index.html'));
+    if(req.query.s === secret || DEBUG === true)
```
Comment (DavidBruant) [inline core/server/admin.js]: * `PRIVATE.json` only contains a string?
* `DEBUG === true` => `DEBUG` (it's a boolean)

Comment (vallettea) [inline core/server/admin.js]: no private has other fields, yes for the second point

File: core/server/admin.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ var zlib = require('zlib');
 
 var PORT = 4001;
 var DEBUG = process.env.NODE_ENV === "development" ? true : false;
```
Comment (DavidBruant) [inline core/server/admin.js]: no need for `? true : false`, `process.env.NODE_ENV === "development"` is already a boolean

File: core/server/admin.js (hunk 1 above)
Comment (DavidBruant) [inline core/server/admin.js]: then, `=== secret` is weird. `req.query.s` is a string and `secret` an object, it will always be `false`

**Pull request merged**